lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -251,6 +256,134 @@ div.course-wrapper {
         }
       }
     }
+    .proctored-exam-skip-confirm-wrapper {
+      padding: $baseline 30px;
+      background-color: rgb(242, 244, 245);
+      border-left: 5px solid $red;
+      .msg-title {
+        margin-bottom: 12px;
+        font-size: 22px;
+        font-weight: 600;
+      }
+      .msg-content {
+        color: rgb(121, 118, 118);
```

Comment:
Will one of our [preset variables](https://github.com/edx/edx-platform/blob/master/lms/static/sass/base/_variables.scss#L59) work for this instead?